.phrase.yml

```diff
@@ -88,12 +91,12 @@ phrase:
       - file: ./config/locales/phrase-exports/ja.yml
         params:
           locale_id: ja
-      - file: ./config/locales/phrase-exports/ka.yml
```

Comment:
We should update the defaults to match, which means:

- removing config/locales/defaults/ka.yml
- renaming config/locales/defaults/mr-IN.yml
- removing config/locales/defaults/tl.yml

fil.yml and ky.yml are not provided by rails-i18n or we'd be adding those as well.

Reply:
Wait, I'm confused. I thought Translation started using `tl` rather than `fil` on Phrase specifically to take advantage of `config/locales/defaults/tl.yml`?

Reply:
I see. In that case we should rename tl.yml to its new name. Can you also add the rename cases to https://github.com/otwcode/otwarchive/blob/master/config/locales/README.md so we don't lose the changes in future rails-i18n updates?

Reply:
Yup, already planning to! This is not a headache I want to repeat 😆

Reply:
Ugh, one more thought: doesn't renaming these files mean that when we import to Phrase, the defaults will be added to Phrase as fil and mr, rather than updating what's already on Phrase as tl and mr-IN?

Reply:
We don't push the defaults files after #4006.

Reply:
Right, sorry, I'm thinking about the hypothetical catastrophe we mention in the README rather than routine work. I suppose we could just manually rename the files before uploading if it ever happens, though.

Reply:
I see what you mean. In Phrase's UI looks like we can use "Edit Language" to change the name and code. We should try similarly renaming there as well, e.g. `tl` to `fil`, that way the pull settings can now be consistent:
```yml
      - file: ./config/locales/phrase-exports/fil.yml
        params:
          locale_id: fil  
```

Reply:
I'll double check with Translation that that'll work for them.

Reply:
Done on their end and done on my end!